### PR TITLE
Adding Noob Black

### PR DESCRIPTION
**Pull request by rocketeeroof**
This version of Noob Black is the same one that has been modified to fit v5, only that it works for v4.

### CODE REVIEW

File: RLBotPack/Noob_Black/src/bot.py [hunk 1]
```diff
@@ -0,0 +1,528 @@
...
+        begin = time.time()
+        self.boost_pad_tracker.update_boost_status(packet)
+
+        if self.active_sequence is not None and not self.active_sequence.done:
+            controls = self.active_sequence.tick(packet)
+            if controls is not None:
+                return controls
+        # Get the amount of time taken before the next bounce
+        def next_bounce(pos, vel, height, g):
+            def solve_quadratic(a, b, c, ans, side):
```
Comment (NicEastvillage) [inline RLBotPack/Noob_Black/src/bot.py:39]: You could gain many performance improvements by moving these nested functions to the global scope instead.

File: RLBotPack/Noob_Black/src/bot.cfg [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+# Short description of the bot
+description = Noob Bot V3, made with code from the black market.
+As always, mainly designed for 3s.
+
+# Fun fact about the bot
+fun_fact = The bot has a Roblox background.
+
+# Tags
+tags = goalie, teamplay, rumble, hoops
```
Comment (NicEastvillage) [inline RLBotPack/Noob_Black/src/bot.cfg:27]: The goalie tag is reserved for bots that ONLY play goalie. The rumble tag is for bots that use the rumble items.

**Review by NicEastvillage — CHANGES_REQUESTED** [review]
Not only is your bot basically one file, it is also basically one function (with nested ones instead). Impressive.

Maybe your next endeavour should be learning how to structure your bot in multiple files and delete the stuff you don't use. You could also improve your code by getting rid of magic numbers.

I need you to fix your tags before I accept the PR.

**Comment by rocketeeroof** [comment]
The tags have been changed.

**Review by NicEastvillage — APPROVED** [review]

**Pull request merged**